### DISCUSSION TITLE
style load after page show

### DISCUSSION THREAD

**Discussion by sonnv1912** [primefaces/community#335] [PrimeReact] [answered]
I'm using nextjs 13.5.4, with primereact 10.0.2
When i use useRouter to change page everything is fine but when i reload by url or Crt + R then components of prime work not correctly it lose it style until page load successfully it work fine again. Please help me  with this
![Snipaste_2023-10-28_19-58-14](https://github.com/primefaces/community/assets/94950483/df16a672-1a14-49f2-b9c4-3e68e91e18a2)

**Comment by habubey** [comment]
Hi, we are aware of the issue and are actively working on it. We plan to resolve the problem this week.
Best

**Reply by melloware** [reply, marked as answer]
I believe your issue might be FOUC like this: https://github.com/primefaces/primereact/issues/5188

There is a workaround provided in that ticket.

**Reply by sonnv1912** [reply]
Thank you so much
